### ISSUE TITLE
add topics for hydrology

### ISSUE THREAD

**Issue by amilan17** [wmo-im/wis2-topic-hierarchy#138]
> Note: sections marked with * are required.

# Overview of proposal*
_(Description of the amendment(s) requested. Include as much relevant information as possible, such as rationale, purpose, and implementation plans.)_

# Amendment details*
proposal circa Feb 2024, see branch
* prediction
   * hydrological-hazards
      -  short-range-flash
      - long-range-riverine
      - nine-months-hdi
      - three-months-phdi 
      - spei 
   * water-resources-groundwater
      - monthly-well-water-levels
      - three-month-springs-levels
   * water-resources-soil
       - seasonal-soil-moisture
   * water-resources-surface
      * monthly-streamflow
      * stream-stage
      * monthly-reservoir-levels
      * seasonal-snowmelt
   * water-resources 
      * monthly-streamflow
   * space-based-observations
      * [earth-systems](https://github.com/wmo-im/wis2-topic-hierarchy/blob/hydrology/topic-hierarchy/earth-system-discipline/hydrology/space-based-observations/earth-systems/index.csv)
      * space-based-observations
         * soil
           * ...
         * groundwater-quality
           * ...
         * groundwater-quantity	
           * ...
    * subsurface-based-observations
       * water-quantity	
        * water-quality
        * land-atmospheric
        * sediment-surface-water
     * surface-based-observations

# Requestor(s)*
_(Name of individuals, groups, or organizations that are requesting the amendment. Include relevant affiliations and GitHub user names, as applicable.)_

# Consultations
_(Individuals, groups,or organizations that are consulted. Also cite publications or other documents as references.)_

# Describe WIS2 data publication plans associated with the proposal
_(identify relevant datasets, WIS2 Nodes, discovery metadata publication)_

# Comments

**Comment by amilan17** [comment]
@PaoloLeoniIT @disiiwash-- can you help topics for hydrology? I think we want to narrow the list of topics down to the types of datasets that will be initially shared on WIS2...

**Comment by PaoloLeoniIT** [comment]
> @sgrellet @PaoloLeoniIT @disiiwash-- can you help topics for hydrology? I think we want to narrow the list of topics down to the types of datasets that will be initially shared on WIS2...

Yes sure, here you can find the updated file where there are also other variables.  

[Hydrology_Variables_URI_code register WMO_v20240529.docx](https://github.com/user-attachments/files/17631557/Hydrology_Variables_URI_code.register.WMO_v20240529.docx)

**Comment by tomkralidis** [comment]
2024-11-26
- @tomkralidis / @amilan17 to discuss with hydrology experts on WIS2 data publication requirements (consultation phase)

**Comment by disiiwash** [comment]
See the branch for the revised proposed topic for hydrology (https://github.com/wmo-im/wis2-topic-hierarchy/tree/hydrology2)

**Comment by disiiwash** [comment]
> > [@sgrellet](https://github.com/sgrellet) [@PaoloLeoniIT](https://github.com/PaoloLeoniIT) @disiiwash-- can you help topics for hydrology? I think we want to narrow the list of topics down to the types of datasets that will be initially shared on WIS2...
> 
> Yes sure, here you can find the updated file where there are also other variables.
> 
> [Hydrology_Variables_URI_code register WMO_v20240529.docx](https://github.com/user-attachments/files/17631557/Hydrology_Variables_URI_code.register.WMO_v20240529.docx)

@PaoloLeoniIT Here is the attached updated file: 

[Hydrology_Variables_URI_code register WMO_v2025-2026_github.docx](https://github.com/user-attachments/files/20770998/Hydrology_Variables_URI_code.register.WMO_v2025-2026_github.docx)

**Comment by amilan17** [comment]
[https://github.com/wmo-im/wis2-topic-hierarchy/tree/hydrology2/topic-hierarchy/earth-system-discipline/hydrology](https://github.com/wmo-im/wis2-topic-hierarchy/tree/hydrology2/topic-hierarchy/earth-system-discipline/hydrology)

| 1.1.1https://github.com/wmo-im/wis2-topic-hierarchy/blob/hydrology/topic-hierarc[WO1] hy/earth-system-discipline/hydrology/space-based-observations/earth-systems/index.csv |
| --- |
| Name | Description | Source |
| glacier-area | Glacier Area | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_12013 |
| glacier-mass-balance | Glacier mass balance at a point | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_12009 |
| glacier-motion | Glacier motion | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_611 |
| glacier-topography | Glacier topography | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_612 |
| glacier-wide mass-balance | Glacier-wide mass balance | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_12010 |
| ice-sheet-topography | Ice sheet topography | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_613 |
| ice-thickness | Ice thickness | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_725 |
| land-cover | Land cover | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_236 |
| land-surface-temperature | Land surface temperature | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_389 |
| land-surface-topography | Land surface topography | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_237 |
| permafrost-temperature | Permafrost temperature | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_12014 |
| presence-of-snow | Presence of snow | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_12007 |
| snow-cover-extent | Snow cover extent | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_628 |
| snow-depth | Snow depth | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_629 |
| soil-moisture | Soil moisture | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_12277 |
| soil-moisture-surface | Soil moisture at surface | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_528 |
| soil-temperature | Soil temperature | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_596 |
| soil-type | Soil type | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_246 |
| wetland-extent | Wetland extent | https://codes.wmo.int/wmdr/ObservedVariableTerrestrial/_243 |

**Comment by amilan17** [comment]
this issue is superseded by #239
